client/src/styles/legacy/_actionTabs.scss

```diff
@@ -206,13 +206,12 @@ $border: 1px solid darken(#D9D9D9, 15%);
             border: none;
           }
           &.loading {
-            background: transparent url("../images/network-save.gif") no-repeat ($spacer / 2) center;
+            background: transparent url('../../images/network-save.gif') no-repeat ($spacer / 2) center;
```

Comment:
URLs for assets should be relative to the file which references them. In this case that means looking in `client/src/images/` where before this pointed at `client/src/styles/images/`.
I think the old webpack config probably rewrote the links as relative to the scss entrypoint which is just silly.